lib/view_model/wallet_keys_view_model.dart

```diff
@@ -276,7 +291,9 @@ abstract class WalletKeysViewModelBase with Store {
       if (_appStore.wallet!.seed == null &&
           _appStore.wallet!.privateKey != null)
         'private_key': _appStore.wallet!.privateKey!,
-      if (restoreHeightResult != null) ...{'height': restoreHeightResult}
+      if (restoreHeightResult != null) ...{'height': restoreHeightResult},
+      if (_appStore.wallet is MoneroWalletBase) ...{'passphrase': (_appStore.wallet as MoneroWalletBase).passphrase()??''},
+      if (_appStore.wallet is WowneroWalletBase) ...{'passphrase': (_appStore.wallet as WowneroWalletBase).passphrase()??''}
```

Comment:
better compare with wallet type, cause this will be undefined when building Monero.com